res2df/parameters.py

```diff
@@ -10,22 +10,22 @@
 import pandas as pd
 import yaml
 
-from ecl2df.eclfiles import EclFiles
+from res2df.resdatafiles import ResdataFiles
 
 logger = logging.getLogger(__name__)
 
 
 def find_parameter_files(
-    ecldeck_or_eclpath: Union[EclFiles, str, Path], filebase: str = "parameters"
+    deckpath: Union[ResdataFiles, str, Path], filebase: str = "parameters"
 ) -> List[Path]:
     """Locate a default prioritized list of files to try to read as key-value
 
     File extensions .yml, .json and .txt are recognized and will be found in
     current dir, one directory up, and two directories up.
 
     Args:
-        ecldeck_or_eclpath: Either an EclFiles object of
-            an Eclipse output set (only the corresponding path will be used),
+        deckpath: Either a ResdataFiles object of
+            a simulator output set (only the corresponding path will be used),
```

Comment:
I think @berland will have to clarify a bit here, but I think `a simulator output set` is the same as `casepath` elsewhere? In which I think we should be consistent and add another term in the glossary for that. It should be `dirname(f) + "/" + basename(f)` of all output files `f` corresponding to the input file `f.DATA`. Ie. if you have `/my/path/CASE.DATA` then after running the simulator you get the file `/my/path/CASE.INIT` etc. These belong to the "casepath" `/my/path/CASE`.